app/serializers/event_serializer.rb

```diff
@@ -28,4 +29,10 @@ def estimated_distance
 
     Geo.distance_between(user_location, object)
   end
+
+  def event_details
+    return '' if object.pub_desc_long.nil?
+
+    object.pub_desc_long
```

Comment:
```suggestion
    object.pub_desc_long || ''
```
Little bit simpler way to set a default.